harness/tests/src/main/kotlin/godot/tests/Entry.kt

```diff
@@ -129,7 +130,25 @@ class Entry : godot.runtime.Entry() {
                             className = "Unit"
                         }
                 )
+                function(Invocation::parentIsSpatial, ::KtVariant,
```

Comment:
Note to myself: Whoever merges last, needs to change this to annotation registration